std/typecons.d

```diff
@@ -2638,33 +2638,36 @@ unittest
 
 /**
 $(D WhiteHole!Base) is a subclass of $(D Base) which automatically implements
-all abstract member functions as throw-always functions.  Each auto-implemented
-function fails with throwing an $(D Error) and does never return.  Useful for
-trapping use of not-yet-implemented functions.
+all abstract member functions as functions that always fail. These automatically
+implemented function simply throw an $(D Error) and never return. `Whitehole` is 
+useful for trapping the use of class member functions that haven't yet been implemented.
 
 The name came from
 $(WEB search.cpan.org/~mschwern/Class-_WhiteHole-0.04/lib/Class/_WhiteHole.pm, Class::_WhiteHole)
 Perl module by Michael G Schwern.
 
-Example:
---------------------
-class C
-{
-    abstract void notYetImplemented();
-}
-
-void main()
-{
-    auto c = new WhiteHole!C;
-    c.notYetImplemented(); // throws an Error
-}
---------------------
+Params:
+    Base = A non-final class from which `WhiteHole` can inherit.
```

Comment:
Then perhaps "A non-final class which `WhiteHole` is able to inherit from" would be better? With its current phrasing, it's possible to interpret it as "might inherit". (that's actually how I first interpreted it)